### ISSUE TITLE
Problemer med å motta møteinnkallelser

### ISSUE THREAD

**Issue by weblady** [webrtd/website#126]
Mail sendt 11.06:
Jeg har hatt kontakt med ei dame som har hatt problemer med å sende ut møteinnkallelser fra intron. Hun sendte ut en møteinnkallelse i går formiddag, denne kom frem til en del av medlemmene i klubben, men ikke til alle. Hun mottok også mailen under hvor det står hvilke adresser den ikke ble sendt til, hvor en av de er hennes egen yahoo-adresse. Jeg har testet med å sende en vanlig mail til denne adressa, og da kom mailen fram. Mailadressene som mailen ikke kom fram til er vanlige yahoo- og gmailadresser, og det er reelle adresser, så jeg kan ikke se noen logikk i hvorfor disse ikke kom fram? Hun trodde at det var gmail og yahoo som ikke takler mailer fra intron, men jeg har gmail-adresse selv, og jeg mottok møteinnkallelse som jeg selv sendte ut til min klubb i går morges. Så jeg klarer ikke å se logikken i dette. Har du mulighet til å finne utav dette?

Mailen under er altså sendt ut til LC 48 Bergen, fra Christine Dumben, cdumben@yahoo.com

Hun får undelivered mail to sender i retur.

**Comment by weblady** [comment]
Dette er fremdeles et problem, og jeg har forsket litt mer på dette:

Hun har lagt inn ei yahoo-adresse som mailadresse på sin profil, og når hun sender ut mail/møteinnkallelser fra intron, så mottar ikke personer med yahoo- og gmail-adresser disse mailene, men alle andre mottar de. Jeg har logget meg  inn med hennes profil og fikk samme problem, men innlogget som megselv med mi gmail-adresse så var det ingen problemer å sende ut deres møteinnkallelser. 

Hun mottar undelivered mail i retur når hun ikke får sendt ut mail til gmail- og yahooadresser. Vet du hva dette kan komme av?

**Comment by webrtd** [comment]
Hvad står der i "undelivered mail" teksten? Det kan betyde mange ting

----------------------------------------------------
Med venlig hilsen / Best regards
Rasmus Kaae, Formand 2014-2015, RT111 Favrskov <http://www.rt111.dk>
National Webmaster since 2011, Round Table Denmark <http://www.rtd.dk>

----------------------------------------------------

2015-11-10 10:01 GMT+01:00 weblady <notifications@github.com>:

**Comment by weblady** [comment]
Dette hadde jeg kun sendt deg på mail, men legger inn her også:

Hei

Her forsøkte jeg å sende mail til megselv innlogget som Christine Dumben, og hun fikk mailen under i retur....

Det samme skjer med møteinnkallelser. 

Jeg ser at jeg hadde lagt inn sak om dette i Github i juni i år, men uten å ha fått noe svar på det. Oppdaterte #126 i github med ny info nå nettopp. 


Silje  

From: Christine Dumben [mailto:cdumben@yahoo.com] 
Sent: fredag 6. november 2015 15.16
To: Weblady LC Norge <web@lcnorge.no>
Subject: Fw: Undelivered Mail Returned to Sender

Og den ser slik ut....
Sendt fra Yahoo Mail på Android

Den fre.. 6. nov. 6, 2015 kl. 15:14, Mail Delivery System
<MAILER-DAEMON@vm0022.enterprisecloud.nu> skrev:
This is the mail system at host vm0022.enterprisecloud.nu.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                  The mail system

<ssalte@gmail.com>: host gmail-smtp-in.l.google.com[64.233.165.26] said:
    550-5.7.1 Unauthenticated email from yahoo.com is not accepted due to
    domain's 550-5.7.1 DMARC policy. Please contact administrator of yahoo.com
    domain if 550-5.7.1 this was a legitimate mail. Please visit 550-5.7.1
    https://support.google.com/mail/answer/2451690 to learn about DMARC 550
    5.7.1 initiative. l198si251818lfe.116 - gsmtp (in reply to end of DATA
    command)
Tester å sende beskjed fra Christine til Silje...

**Comment by webrtd** [comment]
Ja. Dem der styrer jeres domæne skal konfigurere en SPIF record

----------------------------------------------------
Med venlig hilsen / Best regards
Rasmus Kaae, RT111 Favrskov <http://www.rt111.dk>
National Webmaster since 2011, Round Table Denmark <http://www.rtd.dk>

----------------------------------------------------

2015-11-17 12:07 GMT+01:00 weblady <notifications@github.com>:

**Comment by weblady** [comment]
Okei, så da vil det bli åpnet opp for å sende fra yahoo fra vår intron? Forstår jeg det riktig da? At New Media som vi har domenet hos må gjøre denne jobben?

**Comment by webrtd** [comment]
det skal konfigureres som på rtd.dk, de kan prøve at slå det op

----------------------------------------------------
Med venlig hilsen / Best regards
Rasmus Kaae, RT111 Favrskov <http://www.rt111.dk>
National Webmaster since 2011, Round Table Denmark <http://www.rtd.dk>

----------------------------------------------------

2015-11-17 12:25 GMT+01:00 weblady <notifications@github.com>:
